packages/test-base/src/androidAndroidTest/kotlin/io/realm/kotlin/test/shared/RealmConfigurationTests.kt

```diff
@@ -156,7 +156,7 @@ class RealmConfigurationTests {
 
     @Test
     fun directory_createIntermediateDirs() {
-        val realmDir = tmpDir + "/my/intermediate/dir"
+        val realmDir = tmpDir + "${PATH_SEPARATOR}my${PATH_SEPARATOR}intermediate${PATH_SEPARATOR}dir"
```

Comment:
Could also just use `setOf(tmpDir, ....).joinToString() { }` another time. Or maybe even an utility `pathOf(...)`. Would be a little easier to read.

Reply:
Ah yeah, good idea 👍